pkg/virt-controller/services/template_test.go

```diff
@@ -3615,17 +3634,8 @@ var _ = Describe("Template", func() {
 			claimMap := map[string]*kubev1.PersistentVolumeClaim{}
 			pod, err := svc.RenderHotplugAttachmentPodTemplate([]*v1.Volume{}, ownerPod, vmi, claimMap, false)
 			Expect(err).ToNot(HaveOccurred())
-
-			Expect(pod.Spec.Containers[0].Resources).To(Equal(kubev1.ResourceRequirements{
-				Limits: kubev1.ResourceList{
-					kubev1.ResourceCPU:    resource.MustParse("100m"),
-					kubev1.ResourceMemory: resource.MustParse("80M"),
-				},
-				Requests: kubev1.ResourceList{
-					kubev1.ResourceCPU:    resource.MustParse("100m"),
-					kubev1.ResourceMemory: resource.MustParse("80M"),
-				},
-			}))
+			verifyPodRequestLimits1to1Ratio(pod)
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
What is the expect about? did you mean to return err from verifyPodRequestLimits1to1Ratio?

Reply:
I moved all the testing into a function, and forgot to remove that.